src/dataDisplay/divider/divider.jsx

```diff
@@ -36,7 +36,7 @@ Divider.propTypes = {
     inverse: PropTypes.bool,
     relaxed: PropTypes.oneOfType([
         PropTypes.bool,
-        PropTypes.oneOf([ 'very' ])
+        PropTypes.oneOf([ 'very' ]),
```

Comment:
Possibly doesn't matter and outside the scope of this PR, but since you were missing with this line... 😁 
I believe our ESLINT rules dictate no spaces between array brackets and the contents.  This should be:
```JavaScript
PropTypes.oneOf(['very']),
```

Reply:
fixed https://github.com/saddlebackdev/react-cm-ui/pull/264/commits/291bb41c658ac648f10212380f6d8b5bf96ce935#diff-1c42a2fcc5480c08023362ce4d5838fc11bbc96b6d1237382c9120913da1cc23R39